RingGrids/src/field.jl

```diff
@@ -517,6 +527,7 @@ end
 Adapt.adapt_structure(to, field::AbstractField) = Adapt.adapt(to, field.data)
   
 Architectures.architecture(field::AbstractField) = architecture(field.grid)
+Architectures.on_architecture(field::AbstractField, x) = on_architecture(architecture(field), x)
```

Comment:
@maximilian-gelbrecht added this as it felt convenient to just get a field on the architecture of another field